### PR TITLE
first steps

### PR DESCRIPTION
**Pull request by denisergashbaev**
Conflicts:
	templates/guess_the_painter.html
	templates/layout_logged_in.html

### CODE REVIEW

**Comment by denisergashbaev** [comment]
@Jeronics, take a look and take care of the rest please. you would need to find out how to pass arguments from within the subtemplates (like login/register) to the above layout in order to set the title attribute accordingly (right now it is static due to the refactoring)

**Comment by Jeronics** [comment]
I don't get it.. What was the problem before? The arguments WERE passed from to the subtemplates! I'll try to show you what I mean!

**Comment by Jeronics** [comment]
Denis!! Merge please! I added head and body to all files!

File: templates/login.html [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+            <div class="metanav">
+            </div>
+            {% for message in get_flashed_messages() %}
+                <div class="flash">{{ message }}</div>
+            {% endfor %}
+            <header>Login</header>
+            <form name="form" method="post" action="">
+                <p><input type="text" name="username">
+
+                <p><input type="text" name="password">
```
Comment (denisergashbaev) [inline templates/login.html]: password field should be of type password

File: templates/layout_logged_in.html [hunk 2]
```diff
@@ -1,22 +1,10 @@
...
-<script src="{{ url_for('static', filename='js/app/app.js') }}"></script>
-<div class="page">
-    <h1>PaintingTrivia</h1>
-    {% for message in get_flashed_messages() %}
-        <div class="flash">{{ message }}</div>
-    {% endfor %}
-    {% block body %}{% endblock %}
-</div>
\ No newline at end of file
+<html>
```
Comment (denisergashbaev) [inline templates/layout_logged_in.html]: could you please merge layout and layout_logged_in into one file (you need to check in the logged in version if some flag in session is set)

File: templates/main_menu.html [hunk 3]
```diff
@@ -1,13 +1,16 @@
 {% extends "layout_logged_in.html" %}
 {% block body %}
-<head>
-    <meta charset="UTF-8">
-    <title>TriviArt</title>
-</head>
 <body>
-    <header>Choose the Game</header>
+    {% include "login_register_segment.html"%}
```
Comment (denisergashbaev) [inline templates/main_menu.html]: good!

**Comment by denisergashbaev** [comment]
@Jeronics  see my comments. Also, please check if the pages validate now here https://validator.w3.org/#validate_by_input

File: templates/login.html (hunk 1 above)
Comment (Jeronics) [inline templates/login.html]: Fixed!

File: templates/layout_logged_in.html (hunk 2 above)
Comment (Jeronics) [inline templates/layout_logged_in.html]: I'm on it!

**Comment by Jeronics** [comment]
I'll change the keynames later! This is done! Close the pull request!!
I have another to push!

**Pull request merged**